src/PrestaShopBundle/Translation/Provider/Type/BackType.php

```diff
@@ -25,21 +24,32 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-namespace PrestaShopBundle\Translation\Provider;
+declare(strict_types=1);
 
-use PrestaShopBundle\Translation\Exception\UnsupportedModuleException;
+namespace PrestaShopBundle\Translation\Provider\Type;
 
 /**
- * Defines that this provider may need information from module.
+ * Properties container for 'back' type of translation.
  */
-interface UseModuleInterface
+class BackType extends AbstractCoreType
```

Comment:
Suggestion: "Back" is ambiguous ; either put the full name BackOffice or explain what it means in phpDoc 😉